app/helpers/application_helper/toolbar/cloud_volume_center.rb

```diff
@@ -7,6 +7,22 @@ class ApplicationHelper::Toolbar::CloudVolumeCenter < ApplicationHelper::Toolbar
                    t,
                    :items => [
                      button(
+                       :cloud_volume_backup_create,
+                       'pficon pficon-volume fa-lg',
+                       t = N_('Create a Backup of this Cloud Volume'),
+                       t,
+                       :klass     => ApplicationHelper::Button::VolumeBackupCreate,
```

Comment:
[...](https://github.com/ManageIQ/manageiq/pull/11639/files#r83563934), and you can use it like this:

```ruby
:klass => ApplicationHelper::Button::GenericFeatureButtonWithDisable,
:options => {:feature => :backup_create},
```